src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -940,6 +948,15 @@ bool FancyZones::IsNewWorkArea(GUID virtualDesktopId, HMONITOR monitor) noexcept
     return true;
 }
 
+bool FancyZones::IsSplashScreen(HWND window)
+{
+    TCHAR splashString[] = L"MsoSplash";

```

Comment:
let's use `wchar_t` types directly, since we don't support `TCHAR` anywhere.

Reply:
I'm strongly against hard-coding behavior for any particular application

Reply:
@ivan100sic we could have a more general `ShouldSkipWindowPlacement` or something. IIRC, we've had a similar application list before, perhaps @enricogior can elaborate.

Reply:
@ivan100sic  This isn't specifi application behaviour. MsoSplash class is widely used throughout Microsoft application for loading screens. I think it's ok to skip those

Reply:
Okay, for now, let's leave it like this.